tests/ValidatorTest.php

```diff
@@ -69,4 +80,21 @@ public function testInvalidEncodingInClassName()
         $this->expectExceptionMessage(sprintf('"%sController" is not a UTF-8-encoded string.', chr(0xA6)));
         Validator::validateClassName(mb_convert_encoding('ŚController', 'ISO-8859-2', 'UTF-8'));
     }
+
+    public function testValidateNotBlank()
+    {
+        $this->assertEquals('testString', Validator::notBlank('testString'));
+    }
+
+    /**
+     * @dataProvider inValidNotBlankDataProvider
+     *
+     * @param string|null $string
+     */
+    public function testInvalidNotBlank($string)
+    {
+        $this->expectException(RuntimeCommandException::class);
+        $this->expectExceptionMessage('This value cannot be blank.');
+        Validator::notBlank($string);
+    }
```

Comment:
```suggestion
    public function testInvalidNotBlank()
    {
        $this->expectException(RuntimeCommandException::class);
        $this->expectExceptionMessage('This value cannot be blank.');
        Validator::notBlank();

        $this->expectException(RuntimeCommandException::class);
        $this->expectExceptionMessage('This value cannot be blank.');
        Validator::notBlank('');
    }
```

As there is no other dataProvider, I would to that to keep it simple. WDYT ?